src/test/java/de/uhd/ifi/se/decision/management/jira/extraction/codesummarizer/TestCodeSummarizer.java

```diff
@@ -48,12 +48,12 @@ public void testJiraIssueKeyNull() {
 	public void testJiraIssueKeyEmpty() {
 		assertEquals("", summarizer.createSummary(""));
 	}
-
+/*
```

Comment:
instead of out commenting, you can use @Ignore annotation to skip the test until it gets fixed